solutions/swb-reference/end-to-end-tests/cypress/support/commands.ts

```diff
@@ -0,0 +1,90 @@
+/// <reference types="cypress" />
+
+// ***********************************************
+// This example commands.ts shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+//
+// declare global {
+//   namespace Cypress {
+//     interface Chainable {
+//       login(email: string, password: string): Chainable<void>
+//       drag(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       dismiss(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
+//     }
+//   }
+// }
+import {
+  ADMIN_PASSWORD_PROPERTY,
+  ADMIN_USER_PROPERTY,
+  BASE_URL_PROPERTY,
+  COGNITO_DOMAIN_NAME_PROPERTY
+} from './constants';
+
+Cypress.on(
+  'uncaught:exception',
+  (err) => !err.message.includes('ResizeObserver loop completed with undelivered notifications') //known issue with cypress
```

Comment:
Do u happen to have a link to where this bug is reported?

Reply:
link added to code for more info
https://github.com/quasarframework/quasar/issues/2233

Reply:
Oh can u add this link as a comment to the code itself on line 48?

Reply:
I did